### PR TITLE
ACM-12428-App-counts-to-use-new-backend-APls

### PR DESCRIPTION
**Pull request by jeswanke**
application counts are now provided by a new backend api: /aggregate/statuses so that they include all; apps in a big environment

Signed-off-by: John Swanke <jswanke@redhat.com>

### CODE REVIEW

File: frontend/src/App.css [hunk 1]
```diff
@@ -8,8 +8,10 @@
...
 #webpack-dev-server-client-overlay,
 #react-refresh-overlay {
-  overflow: scroll;
-  height: 20% !important;
-  top: 80% !important;
+  height: 75px !important;
+  min-height: 75px !important;
+  left: 50% !important;
+  top: 0px !important;
+  width: 400px !important;
```
[the remark this answers is not in the document]
Comment (jeswanke) [reply frontend/src/App.css:15]: oops

File: backend/src/routes/aggregators/statuses.ts [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+  })
+  req.on('end', async () => {
+    const body = chucks.join()
+    const { clusters = [] } = JSON.parse(body) as IRequestStatuses
+    let items = getItems()
+
+    // should we filter count by provided cluster names
+    if (clusters.length) {
+      items = items.filter((item) => {
+        return clusters.some((value: string) => item.transform[3].indexOf(value) !== -1)
```
[the remark this answers is not in the document]
Comment (jeswanke) [reply backend/src/routes/aggregators/statuses.ts:39]: fixed

File: frontend/src/routes/Home/Overview/overviewDataFunctions.tsx [hunk 3]
```diff
@@ -145,65 +145,31 @@ export function getNodeSummary(filteredClusters: Cluster[], t: TFunction<string,
...
-    subscriptions
-  ).map((app) => getApplicationType(app, t))
   const typeTotals: Record<string, number> = {}
-  appTypes.forEach((app) => {
-    if (typeof typeTotals[app] === 'undefined') {
-      typeTotals[app] = 1
-    } else {
-      typeTotals[app] += 1
-    }
+  Object.keys(filterCounts.type).forEach((type) => {
```
Comment (celdrake) [inline frontend/src/routes/Home/Overview/overviewDataFunctions.tsx:170]: Hi @jeswanke 
I believe there could be a bug here, as we're now integrating the Flight Control into the Overview page, we noticed that when switching the Fleet view, an error occurs and it seems to point to this line. In the debugger, I can see that `filterCounts` is just an empty object, with no `type` prop which breaks it.

overview-switch

Comment (zlayne) [reply frontend/src/routes/Home/Overview/overviewDataFunctions.tsx:170]: Hey @celdrake , John fixed this issue here: https://github.com/stolostron/console/pull/3876.

**Pull request merged**